### PR TITLE
Bump aiohttp versions

### PR DESCRIPTION
**Pull request by adamruzicka**
Will need to be cherry-picked into to 0.6 branch

Addresses #230

### CODE REVIEW

**Comment by adamruzicka** [comment]
```
$ poetry install
Installing dependencies from lock file
Warning: The lock file is not up to date with the latest changes in pyproject.toml. You may be getting outdated dependencies. Run update to update them.

[RuntimeError]
Unable to retrieve the package version for /tmp/pypoetry-git-receptor658yw2jj

$ poetry update
Updating dependencies
Resolving dependencies... (3.4s)

[RuntimeError]
Unable to retrieve the package version for /tmp/pypoetry-git-receptorv259_6n7
```
What am I doing wrong?

File: pyproject.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ repository = "https://github.com/project-receptor/receptor"
 version = "1.0.0"
 
 [tool.poetry.dependencies]
-aiohttp = "^3.6.2"
+aiohttp = ">= 3.6.2, < 4.0"
 prometheus_client = "^0.7.1"
```
Comment (evgeni) [inline pyproject.toml]: To solve my pains, this also needs to allow `0.8.z`.

File: pyproject.toml [hunk 2]
```diff
@@ -30,7 +30,7 @@ receptor-affinity = { git = "https://github.com/project-receptor/affinity.git" }
 tox = "^3.14.5"
 yamllint = "^1.20.0"
 black = "^19.10b0"
-psutil = "^5.7.0"
+psutil = "~5.7.0"
```
Comment (evgeni) [inline pyproject.toml]: for the pulp3 usecase, we don't care about dev-deps

**Comment by evgeni** [comment]
and to close out the "Unable to retrieve the package version for /tmp/pypoetry-git-receptor658yw2jj" issue: this is because `receptor` refers to `affinity` and `affinity` refers to `receptor` and this seems to confuse the hell out of `poetry`

**Review by matburt — APPROVED** [review]

**Pull request merged**